crates/l3/appchain-core-contract-client/src/interfaces/operator.rs

```diff
@@ -0,0 +1,103 @@
+use color_eyre::{eyre::eyre, Result};
+use appchain_utils::LocalWalletSignerMiddleware;
+use appchain_utils::{call_contract, invoke_contract};
+use starknet_accounts::ConnectedAccount;
+use starknet_core::types::{FieldElement, InvokeTransactionResult};
+use starknet_providers::{
+    jsonrpc::{HttpTransport, JsonRpcClient},
+    Provider,
+};
+
+pub struct Operator<'a> {
+    signer: &'a LocalWalletSignerMiddleware,
+    address: FieldElement,
+}
+
+impl<'a> Operator<'a> {
+    pub fn new(address: FieldElement, signer: &'a LocalWalletSignerMiddleware) -> Self {
+        Self { signer, address }
+    }
+
+    fn provider(&self) -> &JsonRpcClient<HttpTransport> {
+        self.signer.provider()
+    }
+
+    pub async fn register_operator(
+        &self,
+        new_operator: FieldElement,
+    ) -> Result<InvokeTransactionResult> {
+        invoke_contract(
+            &self.signer,
+            self.address,
+            "register_operator",
+            vec![new_operator.into()],
+        )
+        .await
+    }
+
+    pub async fn unregister_operator(
+        &self,
+        removed_operator: FieldElement,
+    ) -> Result<InvokeTransactionResult> {
+        invoke_contract(
+            &self.signer,
+            self.address,
+            "unregister_operator",
+            vec![removed_operator.into()],
+        )
+        .await
+    }
+
+    pub async fn is_operator(&self, operator: FieldElement) -> Result<bool> {
+        let provider = self.provider();
+        let values =
+            call_contract(provider, self.address, "is_operator", vec![operator.into()]).await?;
+
+        values.first()
+            .map(|value| value.to_string() != "0")
```

Comment:
`value.to_string() != "0"` don't need to convert to string, you can compare to `FieldElement::ZERO`

Or even better, call `is_zero`

Reply:
fixed